test/evergreen.yml

```diff
@@ -4014,8 +4014,8 @@ buildvariants:
       WT_TOPDIR=$(git rev-parse --show-toplevel)
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
-    upload_source_dir: mongo-tests/largescale/many-collection/dbpath/diagnostic.data
-    upload_filename: diagnostic.data.tgz
+    upload_source_dir: mongo-tests/largescale/many-collection
```

Comment:
Would this lead to uploading the whole database, which will be several GBs worth large.

Reply:
You're right, I will need to make some changes on mongo-test end to create a folder that only contains what we need. It currently exists but it is based on the date which could lead to issues.

Reply:
@sulabhM, I have created a dedicated folder, see https://github.com/wiredtiger/mongo-tests/commit/6fe666dab4882e751b0bfa3cd947f4a3fca4029f.
In our case, it will be `many-collection-artifacts`.